src/elm/Eos.elm

```diff
@@ -312,7 +342,7 @@ symbolFromString str =
     in
     case ( maybeSymbolCode, maybePrecision ) of
         ( Just symbolCode, Just precision ) ->
-            if String.length symbolCode == 3 || String.length symbolCode == 4 then
+            if String.all Char.isAlpha symbolCode && (String.length symbolCode == 3 || String.length symbolCode == 4) then
```

Comment:
oh right, good, since we are here, the actual max length for symbols is 7

Reply:
Can we safely change that here or is there somewhere else we need to change as well?

Reply:
not sure how we parse symbols on the frontend, but probably we will have no problems